packages/runtime/container-runtime/src/test/types/validateContainerRuntimePrevious.generated.ts

```diff
@@ -382,6 +382,7 @@ declare type current_as_old_for_Function_detectOutboundReferences = requireAssig
  * typeValidation.broken:
  * "Function_isRuntimeMessage": {"backCompat": false}
  */
+// @ts-expect-error compatibility expected to be broken
```

Comment:
@CraigMacomber - I expected this to switch to the "Removed" modality, not merely saying it is incompatible.  Did something change?  I walked Mario through making the change and building locally, not sure if there's some secret incantation we missed, or what. Do you know?